app/services/analysis/gwas/manhattan_plot.rb

```diff
@@ -0,0 +1,89 @@
+class Analysis
```

Comment:
@Nuanda NOTE TO SELF: this is where I stopped reading the code (has to go home). I'll start again from here.